### ISSUE TITLE
Fig breaks remote Git operations (clone, pull, push)

### ISSUE THREAD

**Issue by geggo98** [withfig/fig#2033]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.0 (21G115)

### Expected behaviour

Git integrations in VS Code, WebStorm and Obsidian work without errors.

### Actual behaviour

Git crashes with:

```
vdollar_percent_expand: env var ${FIGTERM_SESSION_ID} has no value
/Users/xxx.fig/ssh line 2: Bad forwarding specification.
/Users/xxx/.fig/ssh: terminating, 1 bad configuration options
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

### Steps to reproduce

Run Git pull in VS Code, WebStorm or Obsidian on a repository that has a remote with a SSH URL (instead of HTTPS)

### Environment

```yaml
fig-details:
  - 2.7.7
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 12.6.0 (21G115)
environment:
  - shell: /nix/store/4bxr2dp87vdqxqb0x989lpzj27703a3a-fish-3.4.1/bin/fish
  - terminal: iterm
  - cwd: /Users/stefan.schwetschke/.ssh
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - TERM: xterm-256color
    - SHELL: /bin/zsh
    - FIG_SET_PARENT: w0t4p0:CFB0EFE2-ABC9-47D3-A924-339CC9814034
    - PATH: /Users/stefan.schwetschke/.rd/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/stefan.schwetschke/.deno/bin:/run/current-system/sw/bin:/Users/stefan.schwetschke/.iterm2:/Users/stefan.schwetschke/.fig/bin:/Users/stefan.schwetschke/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/run/current-system/sw/bin/:/usr/local/MacGPG2/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/stefan.schwetschke/.nix-profile/bin:/nix/var/nix/profiles/default/bin
    - FIG_PARENT: 
    - FIG_HOSTNAME: FVFFQ2E2Q05N
    - FIG_TERM: 2.7.7
    - FIGTERM_SESSION_ID: a5ee66fa-d814-4fe7-99f1-957af66a6017
    - FIG_PID: 99575
```

**Comment by geggo98** [comment]
Might relate to https://github.com/withfig/fig/issues/2030. At least the error message seems to be similar.

**Comment by geggo98** [comment]
Commenting out the following in .ssh/config fixed the issue:

```
# Fig ssh integration. Keep at the bottom of this file.
Match all
  Include ~/.fig/ssh
````

**Comment by geggo98** [comment]
The workaround from #2030 did not fix the issue. Only manually editing the ssh config file had an effect. So it seems to be a separate issue, beside the similar error message.

**Comment by grant0417** [comment]
This should be resolved by 2.7.8 which is live
